Makefile

```diff
@@ -1,15 +1,21 @@
 # Go parameters
 GOCMD=go
+GOPATH ?= ~/go/bin
```

Comment:
You might confuse this with PATH? setting GOPATH like this will break my environment.
```suggestion
```

Reply:
No, not really and isn't this just setting it if it isn't already set? Maybe though it should have been `GOPATH ?= ~/go` instead?

Reply:
No PATH would setting would be useless here, `go` is already in the path naturally or nothing in the make script would have worked.